test/unit/kernel/src/KernelWithOmpSchedule.cpp

```diff
@@ -76,7 +77,7 @@ struct KernelWithTraitOmpSchedule : KernelWithOmpScheduleBase
 struct KernelWithMemberAndTraitOmpSchedule : KernelWithOmpScheduleBase
 {
     // Set to be different from expected so that it this is used the test would fail
```

Comment:
it -> if

Reply:
And in addition to it, this comment was outdated at all, will remove